EF-Adapter/CasbinDbAdapter.cs

```diff
@@ -65,12 +82,26 @@ public void AddPolicy(string sec, string ptype, IList<string> rule)
 
         #region async operations
 
+        /// <inheritdoc />
+        public async Task AddPoliciesAsync(string sec, string ptype, IEnumerable<IList<string>> rules)
+        {
+            await Task.Run(() => AddPoliciesAsync(sec, ptype, rules));
```

Comment:
Is it an infinite loop?

Reply:
@xcaptain Yes, the test just seems to pass but running actually.

Reply:
well discovered i will change that and create a test to increase the coverage